.github/workflows/instrumentation_tests.yml

```diff
@@ -1,5 +1,6 @@
 name: Instrumentation tests
 on:
+  workflow_dispatch:
```

Comment:
We will be able to fire this view the github actions web UI for random branches to test our changes if we want once this is merged.